### ISSUE TITLE
Feature Request: Full text search

### ISSUE THREAD

**Issue by atomi** [classeur/classeur#39]
I'd like to get a list of files containing _search pattern_ using the address bar of my browser.

**Comment by benweet** [comment]
File content search feature is in the road map. Though, it's a big one to implement. It will require to have all the contents indexed on server side with an Elastic Search service or something.

**Comment by hypervalley** [comment]
I agree with atomi, content search is essential. The search execution frequency may not justify indexing. So it would be more simple, mainly if cache is used. The response don't need be fast. This feature maybe can be add to a pro/premium account. Think about.

**Comment by geckolinux** [comment]
I would like to see something similar to this JS extension for Stackedit:
https://github.com/benweet/stackedit/wiki/UserCustom.globalSearch
It works amazingly well, and it works with local storage so that even unauthenticated users can run a global search on all their documents.
